roles/edpm_bootstrap/tasks/packages.yml

```diff
@@ -39,7 +39,7 @@
 - name: Install required packages to bootstrap EDPM
   ansible.builtin.dnf:
     name: "{{ edpm_bootstrap_packages_bootstrap }}"
-    state: present
+    state: latest
```

Comment:
lgtm, if this doesn't break the periodic pipeline, like a clean revert would do

Reply:
After further thought, and some [discussion on slack](https://redhat-internal.slack.com/archives/CQXJFGMK6/p1705590739161989), I think we need to halt on the approach to switching to use state:latest. We need to think carefully about if we want every ansible execution to perform a minor update. I don't think we do, so I'd like to hold this PR due to that for the time being.

If we can fix the specific issue related to decontainerized libvirt and openstack-selinux with an adoption specific fix (I think that's the intent of https://github.com/openstack-k8s-operators/edpm-ansible/pull/539), then let's go with that for now.